tests/scalers/kafka.test.ts

```diff
@@ -0,0 +1,208 @@
+import * as fs from 'fs'
+import * as sh from 'shelljs'
+import * as tmp from 'tmp'
+import test from 'ava';
+
+const defaultNamespace = 'kafka-test'
+const defaultCluster = 'kafka-cluster'
+const timeToWait = 300
+const defaultTopic = 'kafka-topic'
+const defaultKafkaClient = 'kafka-client'
+
+const kafkaClusterYamlFile = tmp.fileSync()
+const kafkaTopicYamlFile = tmp.fileSync()
+const kafkaClientYamlFile = tmp.fileSync()
+const kafkaApplicationYamlFile = tmp.fileSync()
+
+test.before('Set up, create necessary resources.', t => {
+	sh.config.silent = true
+	sh.exec(`kubectl create namespace ${defaultNamespace}`)
+	t.is(
+		0,
+		sh.exec(`kubectl apply -f 'https://strimzi.io/install/latest?namespace=${defaultNamespace}' --namespace ${defaultNamespace}`).code,
```

Comment:
You may want to pin this to a certain version instead of `latest` in case there are breaking changes in the future